docs/tutorial.py

```diff
@@ -50,8 +57,12 @@
 # Although these models do not handle reading or writing data, they do expose the zarr
 # arrays.
 
-zarr_arr = ome_zarr_image.group[multiscales_meta[0].datasets[0].path]
-pprint(zarr_arr)
+print(ome_zarr_image.arrays)
+full_res_path = ome_zarr_image.multiscales[0].datasets[0].path
+zarr_arr = ome_zarr_image.arrays[full_res_path]
```

Comment:
I'm trying to think of ways to improve this, so that users could do this in one line instead of 2. Given that the only information in the raw `datasets` metadata is "what's the array called, and what are its transforms", the first thing that comes to my mind is to model `datasets` as `Mapping[str, tuple[zarr.Array, ValidTransforms]]`, or `tuple[tuple[zarr.Array, ValidTransforms], ...]`. 

I'm guessing almost every time a user accesses the `path` attribute, it will be immediately prior to getting a reference to the zarr array at that path, so we can shortcut that step by putting references to the zarr arrays there already. The `Mapping` option puts the `path` attribute in the key space of the mapping, but you can also always get the path from the `path` attribute of the zarr arrays, and if that's considered fine than maybe the tuple form is simpler (and ordered). Thoughts?

Reply:
(not a blocker for this PR)

Reply:
How about adding an `.array` property to `Dataset`, that returns a `zarr.Array` object at the path stored in that dataset? So you could do this instead:

```python
zarr_arr = full_res_path = ome_zarr_image.multiscales[0].datasets[0].array
```

Reply:
I thought more about this, and I don't think what I suggested above is possible, because it would require a reference to the zarr Group, which `Dataset` doesn't have.